content/nomos/updates/2024-02-05.md

```diff
@@ -0,0 +1,63 @@
+- Updated the [Nomos Tokenomics Design Canvas](https://www.notion.so/Nomos-Tokenomics-Design-Canvas-e62e78ee0f8c4a719d1f6ffb08574241) with details about the "Validators" role. More roles to be entered in the coming weeks.
+- Added the fork choice rule [PR](https://github.com/logos-co/nomos-specs/pull/58) as described in "Ouroboros Genesis: Composable Proof-of-Stake Blockchains with Dynamic Availability". Tests will be added later.
+- "Follower maintains ledger state as it follows the blockchain" [PR](https://github.com/logos-co/nomos-specs/pull/62). Leader coins are now spent when they become slot leaders. The next step is to have the leaderproof produce a new coin that the validator can then use to lead another slot.
+- Merged the "Add header ID and message" [PR](https://github.com/logos-co/nomos-specs/pull/52).
+- Summarized the mathematical analysis work in the Impact on Forks [chapter](https://www.notion.so/Impacts-of-Approximate-Total-Stake-1b9a248b222449c68c70fb645a733b03#843c7c5e84854f458ada2d91813ea5f4) and Reducing Forking [chapter](https://www.notion.so/Impacts-of-Approximate-Total-Stake-1b9a248b222449c68c70fb645a733b03#fa783cb6d21b42809b1c06b2c6d68f07). More precisely, derived the probability of the “forking” event and the “empty slot” event when the approximate value of the total stake is used. Considered the large total stake and finite number of slots scenario to derive the typical number of forks and empty slots expected in the T number of slots. This framework was used to assess impacts of underestimation/overestimation of the total stake on the number of forks and empty slots. For the detailed mathematical analysis, check the [Overleaf document](https://www.overleaf.com/project/656dfacf4929b4a3d6d2ffe5).
+
+### `development`
+
+- No updates, heavily in research.
+
+## `data availability`
+
+### `research`
+
+- [DA Document](https://www.notion.so/DA-Layer-Comparison-Table-5848811f0af042e2b24c10d3cea9b0a8) has been updated with new details based on the team review (specifically around adding more details about Ethereum Danksharding).
+- Merged a mandatory [PR](https://github.com/logos-co/nomos-specs/pull/60) containing a big chunk of primitives from the ETH specification that's a dependency for building our DA specification. It will help us save a good amount of time during our implementation (and testing) of everything.
+
+### `development`
+
+- Started work on the DA Layer Cryptographic Implementation Details [document](https://www.notion.so/DA-Layer-Cryptographic-Implementation-Details-4d8ca269e96d4fdcb05abc70426c5e7c?d=b2cd8254d6b240d89e029e9d7518ba17#99b607ee531340718520a142e281bda9). This is a Python implementation.
```

Comment:
remove the "cryptographic" please, as it is innacurate (it's not just about cryptography)

Reply:
Not a python implementation, an executable spec